be/src/formats/parquet/column_reader.cpp

```diff
@@ -275,6 +275,7 @@ class MapColumnReader : public ColumnReader {
         if (_value_reader == nullptr) {
             value_column->append_default(offsets.back());
         }
+        map_column->remove_duplicated_keys();
 
```

Comment:
How about later materialized that all key is null?

Reply:
let it be a TODO